src/effects.js

```diff
@@ -631,15 +633,25 @@ jQuery.each( {
 } );
 
 jQuery.timers = [];
+var oldLength = 0;
 jQuery.fx.tick = function() {
 	var timer,
 		i = 0,
 		timers = jQuery.timers;
 
 	fxNow = jQuery.now();
+	oldLength = timers.length;
+	for ( ; i < oldLength; i++ ) {
+		var removedTimers = 0;
 
-	for ( ; i < timers.length; i++ ) {
-		timer = timers[ i ];
+		for ( var j = 0; j < lis.length; j++ ) {
```

Comment:
ok ! not a problem.
I would like to know if the second bug about extra processing was handled?
As I wasn't sure about it I did no mark it in the checklist.